tests/test_ekobox/fakepdf/myCT14llo_NF3/myCT14llo_NF3.info

```diff
@@ -0,0 +1,29 @@
+SetDesc: 'fake CT14llo, 1-loop PDF evolution with HOPPET and HOPPET alphas_s running solution. This set has 1 member PDFs. mem=0 --> max NF=3'
```

Comment:
Just for curiosity, there is no way to reuse the ones that we have already in the `banana` test ?

Reply:
It's going to be complicated: every time you want to test you need to download even the `banana` repository, since they are not packaged in `banana`, but only contained in the repo.

In general, I'd really like to avoid duplication, but in this specific case, I'd like tests to be fully self contained, in both `eko` and `banana`, so if we need these PDF files, better to have them in both places.

Actually, given the duplication and the issue with `lhapdf`, I'd like to avoid having PDF files and `lhapdf` at all, if possible. I know that `ekobox` is supposed to produce files that are going to be read by `lhapdf`, but I'm suggesting a replacement:
- let's make unit tests actually *unit*, and move everything that requires `lhapdf` in benchmarks instead (they will run in a container anyhow)

Reply:
If they are in benchmarks, we can even think about packaging the PDF files within the benchmarks container, and reuse that one for both `banana` and `ekomark`, solving the duplication.

Reply:
Avoiding `lhapdf` here (and in ``banana`) is not possible - we actually want to test we can produce LHAPDF files and the only way to find out we can, is by using `lhapdf` - and indeed that is non-trivial, as their parser is picky

Reply:
I'm simply proposing to change section, and call it a benchmark. I'm telling you that running it as a test is not possible, since the python package is not reliable, and it's going to make the whole test suite failing forever, making all the tests useless.